custom_components/battery_notes/data/library.json

```diff
@@ -459,18 +475,6 @@
             "battery_type": "AAA",
             "battery_quantity": 4
         },
-        {
-            "manufacturer": "Netatmo",
-            "model": "Smart Indoor Module",
-            "battery_type": "AAA",
-            "battery_quantity": 2
-        },
-        {
-            "manufacturer": "Netatmo",
-            "model": "Smart Outdoor Module",
-            "battery_type": "AAA",
-            "battery_quantity": 2
-        },
```

Comment:
I can see that you just changed those and the GitHub editor didn't like that. Please ignore this change.

Reply:
Thanks I've resolved this